### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Feature_Create New File_Screen Reader: Narrator/NVDA is not announcing the label names "Kind" and "Source Folder"

### ISSUE THREAD

**Issue by Manisha370** [microsoft/azure-tools-for-java#3573]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-NewFeature; #MAS1.3.1; #ScreenReader; #C+E-H; #JavatoolingforIntelliJ/Eclipse; #Narrator; #NVDA

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:**
OS: Windows 10 Enterprise (Version 1903, Build: 18362.356)
Screen Reader: Narrator, NVDA
MAS Violated: MAS1.3.1

**Repro Steps:**
1.Turn on Screen Reader
2.Open Java EE - Eclipse application
3.Install 'Azure Toolkit for Eclipse' from Marketplace
4.Open Eclipse and install  the Scala plug-in 
5.Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
6.Tab till File-->New-->Project and press enter. Tab till HDInsight and expand it. Select 'Spark on HDInsight (Scala)'. Tab till Next and press enter
7.Tab till Next button and press enter. Press Finish button
8.In Project Explorer, Expand the created scala project right-click src, New-->Other and press enter
9.Tab till Scala Wizards, expand and select 'Scala Object' and press next
10.Tab till the Combo box "Kind" & button "MsdF/src/main/scala" 
11.Verify whether screen reader is announcing the label names or not

**Actual Result:**
Narrator/NVDA is not announcing the label names for the controls "Kind" & "Source Folder" 

**Expected Result:**
Narrator/NVDA should announce the label name along with the selected list item and the role (Ex: Scala Object, Kind Combo box Collapsed; MsdF/src/main/scala, Source Folder button)

**User Impact:**
Screen Reader users will not understand about the controls if label names are not read by them

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug6_Narrator is not announcing the label names Kind and Source Folder.pptx](https://github.com/microsoft/azure-tools-for-java/files/3635997/Bug6_Narrator.is.not.announcing.the.label.names.Kind.and.Source.Folder.pptx)
![Bug6_NVDA is not announcing the label names](https://user-images.githubusercontent.com/51435937/65328909-0763aa80-dbd5-11e9-8bcd-90b59a8d9018.png)

**Comment by t-rufang** [comment]
It is determined by Eclipse IDE. We cannot fix it. And it's out of the toolkit's scope.

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749680
